src/components/avatar/index.js

```diff
@@ -141,14 +140,23 @@ export default class Avatar extends BaseComponent {
     return badgeColor;
   }
 
+  getBadgePosition() {
+    const {size} = this.props;
+    const radius = size / 2;
+    const x = Math.sqrt(radius ** 2 * 2);
+    const y = x - radius;
+    const shift = Math.sqrt(y ** 2 / 2) - 13.5 / 2; // 13.5 = hard coded badge size
```

Comment:
Move the size of the pimple to a const at the top of the file and reuse it. 
this way you don't need to add comments explaining what you're doing...